Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -70,6 +70,7 @@ def print_filename(string, filename)
         print_filename(string, file) unless linked_libraries.empty?
         linked_libraries.each do |lib|
           puts " #{Tty.gray}-->#{Tty.reset} links to #{lib}"
+
```

Comment:
I see; this makes sense. I didn't realize this and (if mentioned) overlooked it in previous comments. I think it's important to take our first few steps in the right direction (like with the error message and URL formatting), but we don't necessarily have to eliminate all direct uses of `Tty` in this PR, though we should consider doing so in the not too distant future. But as usual, naming is hard as is identifying the necessary styles while consolidating the probably too many different styles currently in use.